resources/views/bios/create.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('layout')
+
+@section('content')
+
+    <div class="container">
+        <div class="row">
+            <div class="col-md-6">
+                <h1>Create Bio</h1>
+
+                <ul class="errors">
+                    @foreach ($errors->all() as $message)
+                        <li>{{ $message }}</li>
+                    @endforeach
+                </ul>
+
+                {{ Form::open(array('action' => 'BiosController@store', 'class' => 'new-bio-form')) }}
```

Comment:
hahaha :) greenfield project that's copied from a project I wrote in 2013... https://github.com/mattstauffer/markedstyle/commit/9b1d7117486d86f3e0eb88866138f9336420ae78

:D agreed though